<?xml version="1.0" ?>
<launch>
    <arg name="machine" default="localhost"/>
    <arg name="speech_synthesizer_machine" default="$(arg machine)"/>
    <arg name="audio_player_machine" default="$(arg machine)"/>
    <arg name="use_machine" default="true"/>

    <!-- <param name="robot_name" value="doris" /> -->

    <machine name="localhost" address="localhost" if="$(arg use_machine)"/>

    <rosparam file="$(find butia_speech)/config/ros.yaml" command="load"/>
    <rosparam file="$(find butia_speech)/config/butia_speech_synthesizer.yaml" command="load"/>
    
    <node name="audio_player" pkg="butia_speech" type="audio_player.py" output="screen" machine="$(arg audio_player_machine)"/>

    <node name="speech_synthesizer" pkg="butia_speech" type="speech_synthesizer.py" output="screen" machine="$(arg speech_synthesizer_machine)"/>
</launch>